### PR TITLE
init for front-end

### PR DESCRIPTION
**Pull request by danielborowski**
:)

### CODE REVIEW

File: front-end/gatsby-node.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+const path = require('path');
+const crypto = require('crypto');
+const uuidv1 = require('uuid/v1');
+const contentful = require('contentful');
+const mongoclient = require('mongodb').MongoClient;
+const { Stitch, AnonymousCredential } = require('mongodb-stitch-server-sdk');
+
+let LANGUAGES = [
```
Comment (i80and) [inline front-end/gatsby-node.js]: Stick with `const` unless a binding definitely _must_ be non-constant.

File: front-end/src/components/Step.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    }
+    else if (sectionData.type === 'directive' && sectionData.name === 'tabs') {
+      return <Tabs tabsData={ sectionData } refDocMapping={ this.props.refDocMapping } key={ index } addLanguages={ this.props.addLanguages } activeLanguage={ this.props.activeLanguage } />
+    }
+    else if (sectionData.type === 'code') {
+      return <CodeBlock codeData={ sectionData } key={ index } />
+    }
+    else if (sectionData.type === 'heading') {
+      return (
+        <h3 key={ index }>{ sectionData.children[0].value }
```
Comment (i80and) [inline front-end/src/components/Step.js]: Keep in mind that step titles may contain arbitrary inline formatting

File: front-end/src/components/Section.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      showAllSteps: !this.state.showAllSteps,
+      showAllStepsText: this.state.showAllSteps ? 'Expand All Steps' : 'Collapse All Steps'
+    });
+  }
+
+  render() {
+    return (
+      <div className="section" id={ this.props.sectionData.name }>
+        <h2> 
+          { this.nameMapping[this.props.sectionData.name] }
```
Comment (i80and) [inline front-end/src/components/Section.js]: Not applicable for guides, but remember that section titles may have arbitrary inline formatting

File: front-end/src/components/Step.js (hunk 2 above)
Comment (danielborowski) [reply front-end/src/components/Step.js]: will fix in future

File: front-end/src/components/Section.js (hunk 3 above)
Comment (danielborowski) [reply front-end/src/components/Section.js]: will fix in future :)

**Pull request merged**